core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -150,10 +156,40 @@ var TopicEditorPage = function() {
     });
   };
 
+  this.expectNumberOfQuestionsForSkillWithIndexToBe = function(
+      count, index) {
+    selectSkillDropdown.click();
```

Comment:
Make sure the element is clickable. (If possible assert that the click has worked as expected.)